amxmodx/msgnatives.cpp

```diff
@@ -0,0 +1,208 @@
+// vim: set ts=4 sw=4 tw=99 noet:
+//
+// AMX Mod X, based on AMX Mod by Aleksander Naszko ("OLO").
+// Copyright (C) The AMX Mod X Development Team.
+//
+// This software is licensed under the GNU General Public License, version 3 or higher.
+// Additional exceptions apply. For full license details, see LICENSE.txt or visit:
+//     https://alliedmods.net/amxmodx-license
+
+#include <amxmodx.h>
+#include <messages.h>
+
+static cell AMX_NATIVE_CALL draw_ammo_pickup_icon(AMX *amx, cell *params)
+{
+	enum args { arg_numargs, arg_id, arg_ammoid, arg_amount, arg_reliable };
+
+	int index = params[arg_id];
+
+	if(!check_msg_receiver(amx, index))
+		return 0;
+
+	MESSAGE_BEGIN(get_msg_destination(index, params[arg_reliable] != 0), gmsgAmmoPickup, NULL, index ? TypeConversion.id_to_edict(index) : NULL);
```

Comment:
`nullptr` maybe? Same below.

Reply:
@WPMGPRoSToTeMa Is there an actual difference? I saw that `NULL` is used the most in `amxmodx.cpp` so I decided to go with that one.

Reply:
When we can, we modernize the code.

`nullptr` is a new keyword introduced in C++11. `nullptr`  is meant as a replacement to `NULL` . `nullptr` provides a type-safe pointer value representing an empty (null) pointer.

For new code like that, you should always use `nullptr`.

Reply:
@Arkshine Okay, I changed it.